posthog/hogql/grammar/HogQLLexer.interp

```diff
@@ -743,4 +746,4 @@ mode names:
 0, 0, 2051, 2052, 7, 30, 0, 0, 2052, 452, 1, 0, 0, 0, 2053, 2054, 7, 31, 0, 0, 2054, 454, 1, 0, 0, 0, 2055, 2056, 7, 32, 0, 0, 2056, 456, 1, 0, 0, 0, 2057, 2058, 5, 45, 0, 0, 2058, 2059, 5, 62, 0, 0, 2059, 458, 1, 0, 0, 0, 2060, 2061, 5, 42, 0, 0, 2061, 460, 1, 0, 0, 0, 2062, 2063, 5, 96, 0, 0, 2063, 462, 1, 0, 0, 0, 2064, 2065, 5, 92, 0, 0, 2065, 464, 1, 0, 0, 0, 2066, 2067, 5, 58, 0, 0, 2067, 466, 1, 0, 0, 0, 2068, 2069, 5, 44, 0, 0, 2069, 468, 1, 0, 0, 0, 2070, 2071, 5, 124, 0, 0, 2071, 2072, 5, 124, 0, 0, 2072, 470, 1, 0, 0, 0, 2073, 2074, 5, 45, 0, 0, 2074, 472, 1, 0, 0, 0, 2075, 2076, 5, 36, 0, 0, 2076, 474, 1, 0, 0, 0, 2077, 2078, 5, 46, 0, 0, 2078, 476, 1, 0, 0, 0, 2079, 2080, 5, 61, 0, 0, 2080, 2081, 5, 61, 0, 0, 2081, 478, 1, 0, 0, 0, 2082, 2083, 5, 61, 0, 0, 2083, 480, 1, 0, 0, 0, 2084, 2085, 5, 62, 0, 0, 2085, 2086, 5, 61, 0, 0, 2086, 482, 1, 0, 0, 0, 2087, 2088, 5, 62, 0, 0, 2088, 484, 1, 0, 0, 0, 2089, 2090, 5, 35, 0, 0, 2090, 486, 1, 0, 0, 0, 2091, 2092, 5, 123, 0, 0, 2092, 488, 1, 0, 0, 0, 2093, 2094, 5, 91, 0, 0, 2094, 490, 1, 0, 0, 0, 2095, 2096, 5, 60, 0, 0, 2096, 2097, 5, 61, 0, 0, 2097, 492, 1, 0, 0, 0, 2098, 2099, 5, 40, 0, 0, 2099, 494, 1, 0, 0, 0, 2100, 2101, 5, 60, 0, 0, 2101, 496, 1, 0, 0, 0, 2102, 2103, 5, 33, 0, 0, 2103, 2107, 5, 61, 0, 0, 2104, 2105, 5, 60, 0, 0, 2105, 2107, 5, 62, 0, 0, 2106, 2102, 1, 0, 0, 0, 2106, 2104, 1, 0, 0, 0, 2107, 498, 1, 0, 0, 0, 2108, 2109, 5, 37, 0, 0, 2109, 500, 1, 0, 0, 0, 2110, 2111, 5, 43, 0, 0, 2111, 502, 1, 0, 0, 0, 2112, 2113, 5, 63, 0, 0, 2113, 504, 1, 0, 0, 0, 2114, 2115, 5, 34, 0, 0, 2115, 506, 1, 0, 0, 0, 2116, 2117, 5, 39, 0, 0, 2117, 508, 1, 0, 0, 0, 2118, 2119, 5, 125, 0, 0, 2119, 510, 1, 0, 0, 0, 2120, 2121, 5, 93, 0, 0, 2121, 512, 1, 0, 0, 0, 2122, 2123, 5, 41, 0, 0, 2123, 514, 1, 0, 0, 0, 2124, 2125, 5, 59, 0, 0, 2125, 516, 1, 0, 0, 0, 2126, 2127, 5, 47, 0, 0, 2127, 518, 1, 0, 0, 0, 2128, 2129, 5, 95, 0, 0, 2129, 520, 1, 0, 0, 0, 2130, 2131, 5, 47, 0, 0, 2131, 2132, 5, 42, 0, 0, 2132, 2136, 1, 0, 0, 0, 2133, 2135, 9, 0, 0, 0, 2134, 2133, 1, 0, 0, 0, 2135, 2138, 1, 0, 0, 0, 2136, 2137, 1, 0, 0, 0, 2136, 2134, 1, 0, 0, 0, 2137, 2139, 1, 0, 0, 0, 2138, 2136, 1, 0, 0, 0, 2139, 2140, 5, 42, 0, 0, 2140, 2141, 5, 47, 0, 0, 2141, 2142, 1, 0, 0, 0, 2142, 2143, 6, 260, 0, 0, 2143, 522, 1, 0, 0, 0, 2144, 2145, 5, 45, 0, 0, 2145, 2146, 5, 45, 0, 0, 2146, 2150, 1, 0, 0, 0, 2147, 2149, 8, 33, 0, 0, 2148, 2147, 1, 0, 0, 0, 2149, 2152, 1, 0, 0, 0, 2150, 2148, 1, 0, 0, 0, 2150, 2151, 1, 0, 0, 0, 2151, 2154, 1, 0, 0, 0, 2152, 2150, 1, 0, 0, 0, 2153, 2155, 7, 34, 0, 0, 2154, 2153, 1, 0, 0, 0, 2155, 2156, 1, 0, 0, 0, 2156, 2157, 6, 261, 0, 0, 2157, 524, 1, 0, 0, 0, 2158, 2159, 7, 35, 0, 0, 2159, 2160, 1, 0, 0, 0, 2160, 2161, 6, 262, 0, 0, 2161, 526, 1, 0, 0, 0, 37, 0, 589, 1087, 1801, 1844, 1849, 1855, 1857, 1866, 1868, 1879, 1881, 1886, 1893, 1898, 1902, 1907, 1912, 1916, 1921, 1928, 1934, 1939, 1946, 1951, 1957, 1962, 1964, 1970, 1975, 1982, 1990, 1992, 2106, 2136, 2150, 2154, 1, 6, 0, 0]
```

Comment:
Let's not change this very often 😅 

This is still better committed into the repo, than built each time. "Install java" is a step you'd need to do otherwise, etc.